lib/ansible/modules/packaging/os/yum.py

```diff
@@ -149,7 +149,8 @@
   enable_plugin:
     description:
       - I(Plugin) name to enable for the install/update operation.
-        The enabled plugin will not persist beyond the transaction.
+        The enabled plugin will not persist beyond the transaction. This description does not seem to enough. I tried this for yum-version
```

Comment:
Please update the PR with an improved description rather than posting the question in the PR change. Thank you.

Reply:
> Please update the PR with an improved description rather than posting the question in the PR change. Thank you.

Well I don't the usage of this option at all and I first time tried with yum-version-lock plugin and it did not work out.

Reply:
Ok. We can work an clarifying the description if it is not clear enough. If you have any helpful suggestions as to how it could be less confusing, please share.

Reply:
Well it will be  best if some sample playbook can be given in the bottom just like for other options, or at least in description itself if some plugin example can be given like enable_plugin={xyz plugin}.
Till now information given is very generic.  I will also try to find some good notes about same  if I get it working.